src/DynamoCore/Models/RecordableCommands.cs

```diff
@@ -491,7 +491,7 @@ private static string TryFindFile(string xmlFilePath, string uriString = null)
             internal static OpenFileCommand DeserializeCore(XmlElement element)
             {
                 XmlElementHelper helper = new XmlElementHelper(element);
-                string xmlFilePath = TryFindFile(helper.ReadString("XmlFilePath"), element.OwnerDocument.BaseURI);
+                string xmlFilePath = TryFindFile(helper.ReadString("FilePath"), element.OwnerDocument.BaseURI);
```

Comment:
hmm, a little worried here, won't this stop old xml files from loading correctly as they have a "XmlFilePath" attribute?